app/api/models/user.py

```diff
@@ -154,7 +155,14 @@ def add_models_to_namespace(api_namespace):
 
 login_response_body_model = Model('Login response data model', {
     'access_token': fields.String(required=True, description='User\'s access token'),
-    'expiry': fields.Float(required=True, description='Access token expiry UNIX timestamp')
```

Comment:
I think this could remain still on this PR, until the android app is refactored to not use expiry, I'm not sure now, I think `expiry` is not used for anything on systers/mentorship-android if this is true than this can be removed :)

Reply:
@isabelcosta i will leave it for now and add a TODO comment